test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/general_functions/GF08/gf08t003/TestDescription.java

```diff
@@ -58,7 +58,7 @@
  *      nsk.jvmti.scenarios.general_functions.GF08.gf08t
  *      gf08t003
  *      nsk.jvmti.scenarios.general_functions.GF08.gf08t003
- *      "Registering JNI native method"
  *      jni
+ *      Registering JNI native method
```

Comment:
The motivation for this change isn't obvious to me, and seems less readable than the original code.

Reply:
the reason I had to this change is that jtreg doesn't handle `"` in any special way, so `gf08t003 nsk.jvmti.scenarios.general_functions.GF08.gf08t003 "Registering JNI native method" jni` is passed to `gf08t` as
[`gf08t003`, `nsk.jvmti.scenarios.general_functions.GF08.gf08t003`, `"Registering`, `JNI`, `native`, `method"`, `jni`]. the way I fixed that was by changing the order of `phrase`, `verboseType`, so `verboseType` is now 2nd arg, and `phrase` 3rd and the rest of args joined.

the alternative way would be to teach `gf08t` about `"` and make it join arguments until it gets an even number of `"` and remove them, this, from my PoV, would be a bit more confusing. if you however are of string opinion here, I can go w/ that alternative (or any other if you have more suggestions)

Reply:
So what code used to handle passing quoted arguments? Is this something that PropertyResolvingWrapper used to do?

Reply:
right, `PropertyResolvingWrapper` was joining arguments which are within `"`